libraries/cfstore/scripts/cfstore_test_spec_k64f_gcc.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
Do we have the old mbedOS version of mbedgt -n command implemented (to be able to run a specific test case? Or a way of running a subset of TESTS?